jmclient/jmclient/wallet_utils.py

```diff
@@ -145,15 +145,26 @@ def get_balance(self, include_unconf=True):
     def get_available_balance(self):
         return sum([x.get_available_balance() for x in self.children])
 
+    def get_balances(self, include_unconf=True):
+        return (self.get_balance(include_unconf=include_unconf),
+                self.get_available_balance())
+
     def get_fmt_balance(self, include_unconf=True):
-        available_balance = self.get_available_balance()
-        total_balance = self.get_balance(include_unconf)
+        total_balance, available_balance = self.get_balances(
+            include_unconf=include_unconf)
         if available_balance != total_balance:
             return "{0:.08f} ({1:.08f})".format(available_balance, total_balance)
         else:
             return "{0:.08f}".format(total_balance)
 
+    def get_fmt_balance_json(self, include_unconf=True):
+        total_balance, available_balance = self.get_balances(
+            include_unconf=include_unconf)
+        return {self.balance_key_name: "{0:.08f}".format(total_balance),
+                "available_balance": "{0:.08f}".format(total_balance)}
```

Comment:
```suggestion
        return {self.balance_key_name: "{0:.08f}".format(total_balance),
                "available_balance": "{0:.08f}".format(available_balance)}
```

Haven't tested this yet but I think it should be like this right?

Reply:
Thanks :) Fixed in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/1285/commits/75a93df85a04da086f1416ad0d705a47dec80b4d